src/components/action-menu/action-menu.jsx

```diff
@@ -76,8 +76,8 @@ class ActionMenu extends React.Component {
         // (possibly slow) action is started.
         return event => {
             ReactTooltip.hide();
+            if (fn) fn(event);
```

Comment:
Why is this change made? The point of the forceHide:true => fn(event) => forceHide: false is to allow the menu to close before the (possibly long running) callback is called.